ERCS/erc-7549.md

```diff
@@ -0,0 +1,479 @@
+---
+eip: 7549
```

Comment:
```suggestion
eip: 7589
```
Assigning next sequential EIP/ERC/RIP number.  7549 was already issued to an EIP.
Numbering changed to sequential from 7500 and is no longer the PR number.

Please also update the filename.

Reply:
Thanks @abcoathup 

Fixed the EIP number and the internal links